webpages/signup.php

```diff
@@ -1,157 +1,201 @@
-    <input type="text" name="email" value="<?php echo $email ?>" placeholder="Email"  />
-    <?php echo $emailerr ?>
-    <input type="password" name="pass" placeholder="Password" />
-    <?php echo $passerr ; ?>
-    <input type="password" name="cpass" placeholder="Confirm Password" />
-    <?php echo $pass2err ; ?>
-    <input type="button" name="next" class="next action-button" value="Next" />
-  </fieldset>
-  
-  <fieldset>
-    <h2 class="fs-title">Personal Details</h2>
-    <h3 class="fs-subtitle">We will never sell it</h3>
-    <input type="text" name="fname" value="<?php echo $fname ?>"  placeholder="First Name"  />
-    <?php echo $fnameerr ; ?>
-    <input type="text" name="lname"  value="<?php echo $lname ?>"  placeholder="Last Name"  />
-    <?php echo $lnameerr ; ?>
-    <div class="gender">
-    <input type="radio" class="gen" name="gender" value="male" checked> Male
-  <input type="radio" class="gen" name="gender" value="female"> Female
-  <input type="radio" class="gen" name="gender" value="other"> Other
-</div>
-    <input type="button" name="previous" class="previous action-button" value="Previous" />
-    <input type="submit" name="submit" class="submit action-button" value="Submit"  />
-  </fieldset>
-</form>
-  <script src='http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js'></script>
-<script src='http://cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.3/jquery.easing.min.js'></script>  
-<script src="../js/signup.js"></script>
-</body>
+                  }
+              $conn->close(); 
+              }
+               
+                } 
+                
+                   
+            
+             ?>
+        <div class="navigate">
+            <a class=" nav-head" id="nav-hd" href="../index.html">WebSiteName</a> 
+            <ul class="nav-ri" style="margin-left:-50px; ">
+                <form class="form-inline login-form" action="signup.php">
+                    <input type="text" placeholder="Email or Phone" class=" login-text transp" id="email">
+                    <input type="password" placeholder="password" class=" login-text transp" id="pwd">
+                    <button  type="submit" class="submit-button transp">Login</button>
+                    <br>
+                    <div class="forgot">Forgotten Password ?</div>
+                </form>
+            </ul>
+        </div>
+        <!-- multistep form -->
+        <form id="msform" method="post" action="<?php echo htmlspecialchars($_SERVER["PHP_SELF"]);?>">
+            <!-- progressbar -->
+            <ul id="progressbar">
+                <li class="active">Account Setup</li>
+                <li>Personal Details</li>
+            </ul>
+            <!-- fieldsets -->
+            <fieldset>
+                <h2 class="fs-title">Create your account</h2>
+                <h3 class="fs-subtitle">This is step 1</h3>
+                <input type="text" name="email" value="<?php echo $email ?>" placeholder="Email"  />
```

Comment:
Use input type `email` for frontend validation here.

Reply:
intentionally done. With frontend validation if a user enters wrong input it won't let user to submit the form and as the input box is in first page and submit button is in 2nd page so I won't show the error also.

Reply:
you do it for testing purpose only, that you can also handle through chrome dev tools, when pushing keep this in mind